docs/Translation.md

```diff
@@ -12,27 +12,60 @@ The react-admin interface uses English as the default language. But it also supp
 To handle translations, the `<Admin>` component supports:
 
 - a `locale` prop expecting a string ('en', 'fr', etc), and
-- a `messages` prop, expecting a dictionary object.
+- a `i18nProvider` prop, expecting an `i18nProvider` as explained below.
 
-React-admin ships with the English locale by default. If you want to use another locale, you'll have to install a third-party package. For instance, to change the interface to French, install the `ra-language-french` npm package, then configure the `<Admin>` component as follows:
```

Comment:
React-admin can still work without provider and default to English, right?

Reply:
Yes, the default provider loads the english messages. Not sure if this should be changed to an empty messages object, because the same bundle is loaded in the `TranslationProvider` as default messages.  Having no messages in the defaultProvider feels weird, let me know what you want.

Reply:
Yeah, so I rephrased it a bit, probably should have left the `React-admin ships with the English locale by default.` sentence in there, will change.